examples/amp-next-page.amp.html

```diff
@@ -21,7 +21,6 @@
   <link rel="canonical" href="https://medium.com/p/cb7f223fad86" >
```

Comment:
Can we revert this or is it still needed?

Reply:
I also added an `<amp-iframe>` to the page, but can also remove it since we have two more components.

Reply:
No preference on my side